ui/plugins/pluginpanel.html

```diff
@@ -68,33 +68,34 @@
             </a>
           </li>
           <li>
-            <!--the first 3 items are in the same box-->
-            <ul class="nav navbar-nav">
```

Comment:
This removal isn't new to this PR, but I really couldn't figure out why this `ul` was present for the Pin btn/counter group. I removed it, so the Pin btn is just a regular anchor like log and raw log. Of course it will probably be revealed why after testing on stage. :)